src/fvmcts/fv_mcts_vanilla.jl

```diff
@@ -192,25 +193,27 @@ function maxplus_joint_mcts_planner(solver::FVMCTSSolver,
 
     @assert (node_exploration || edge_exploration) "At least one of nodes or edges should explore!"
 
-    adjmat = coord_graph_adj_mat(mdp)
-    @assert size(adjmat)[1] == n_agents(mdp) "Adjacency Mat does not match number of agents!"
+#=     adjmat = coord_graph_adj_mat(mdp)
+    @assert size(adjmat)[1] == n_agents(mdp) "Adjacency Mat does not match number of agents!" =#
 
-    adjmatgraph = SimpleGraph(adjmat)
+    #adjmatgraph = SimpleGraph(adjmat)
+    adjmatgraph = coordination_graph(mdp)
+    @assert size(adjacency_matrix(adjmatgraph))[1] == n_agents(mdp)
     
     # Initialize full agent actions
     # TODO(jkg): this is incorrect? Or we need to override actiontype to refer to agent actions?
-    all_agent_actions = Vector{actiontype(mdp)}(undef, n_agents(mdp))
+    all_agent_actions = Vector{(actiontype(mdp))}(undef, n_agents(mdp))
     for i = 1:n_agents(mdp)
-        all_agent_actions[i] = get_agent_actions(mdp, i)
+        all_agent_actions[i] = agent_actions(mdp, i)
     end
 
-    mp_stats = MaxPlusStatistics{eltype(init_state)}(adjmatgraph,
+    mp_stats = MaxPlusStatistics{S}(adjmatgraph,
```

Comment:
Good catch! Even in the previous version, this should have been `S` and not `eltype`